tests/goth/assets/goth-config.yml

```
@@ -0,0 +1,62 @@
+# Note: values of keys denoting paths are resolved relative to the directory
+# in which this file is located.
+# The tokens `~` and `~user` are also replaced by the corresponding users's
+# home directory.
+
+docker-compose:
+
+  docker-dir: "docker/"
+
+  build-environment:
+    # TODO:
+    # For now these settings are common to all `yagna` containers.
+    # In future we may want to have nodes running different versions
+    # of `yagna` in the test network.
+
+    # binary-path: ...
+    # deb-path: ...
+    # branch: ...
+    # commit-hash: ...
+    # release-tag: ... (e.g. "latest", "0.6.2")
+
+  compose-log-patterns:
+    ethereum: ".*Wallets supplied."
+    zksync: ".*Running on http://0.0.0.0:3030/.*"
+
+
+key-dir: "keys"
+
+
+web-root: "web-root"
+
+
+node-types:
+  # Each node type is a collection of attributes common to a group of nodes.
+  # Required attributes are "name" and "class".
+
+  - name: "Requestor"
+    class: "goth.runner.probe.RequestorProbe"
+
+  - name: "VM-Wasm-Provider"
+    class: "goth.runner.provider.ProviderProbeWithLogSteps"
```

```suggestion
    class: "goth.runner.provider.ProviderProbe"
```

Once https://github.com/golemfactory/goth/pull/454 is merged. I'm getting more and more skeptical about having these assets duplicated across repos. :(

---

Good point, I've removed the asset files from the repository and instead added a command that creates them every time the tests are run -- see https://github.com/golemfactory/yapapi/pull/281/commits/f16cc5af2c76c3bfff91e38113d1a55e8226956e